portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -116,13 +95,11 @@ private int getUserChoice(HttpServletRequest request) {
     try {
       userChoice = Integer.parseInt(userChoiceString);
     } catch (NumberFormatException e) {
-      System.err.println("Could not convert to int: " + userChoiceString);
       return 5;
```

Comment:
In general [sentinel values](https://en.wikipedia.org/wiki/Sentinel_value) should be things that we never expect to occur. In this case if a 5th option was added at a later point this code would be updated, this is why '-1' is often used since adding new values will never invalidate this code.  

No need to change this immediately - just something to consider.